gnovm/pkg/gnomod/pkg.go

```diff
@@ -5,40 +5,32 @@ import (
 	"io/fs"
 	"os"
 	"path/filepath"
+	"strings"
 )
 
 type Pkg struct {
-	name     string
-	path     string
-	draft    bool
-	requires []string
+	Dir      string   // absolute path to package dir
```

Comment:
Dir here is fine but there should also be a PkgPath. 
Module.PkgPath.

Reply:
We already have `Name string` field for pkg name/path. 

Maybe rename it to Path? (`s/Name/Path`)